components/ui/card.tsx

```diff
@@ -2,16 +2,86 @@ import { colors } from "@/theme";
 import { Flex, FlexProps } from "@chakra-ui/react";
 import React from "react";
 
-const Card = (props: FlexProps) => {
+type CardProps = {
+  variant?: "whiteShadow" | "locked" | "active" | "gradient";
+} & FlexProps;
+
+const Card = (props: CardProps) => {
+  const { variant, ...rest } = props;
+  switch (variant) {
+    case "whiteShadow":
+      return <WhiteShadow {...rest} />;
+    case "locked":
+      return <Locked {...rest} />;
+    case "active":
+      return <Active {...rest} />;
+    case "gradient":
+      return <Gradient {...rest} />;
+    default:
+      return (
+        <Flex
+          bg={"lily-white.100"}
+          borderWidth={1}
+          borderColor={"black"}
+          p={4}
+          flexDir={"column"}
+          {...rest}
+        />
+      );
+  }
+};
+
+const WhiteShadow = ({ ...rest }: FlexProps) => {
+  return (
+    <Flex
+      bg={"white"}
+      shadow={"5px 5px 0 black"}
+      borderWidth={1}
+      borderColor={"black"}
+      p={4}
+      flexDir={"column"}
+      {...rest}
+    />
+  );
+};
+
+const Locked = ({ ...rest }: FlexProps) => {
```

Comment:
Oh and if it's a variant called locked, it should also render the icon, if that's something you still want. Easy way to have in inconsistent UI is to have two switches you need to flick when you want one thing to happen.